docs/diagrams/SortCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,36 @@
+@startuml
+participant ":Main" as Main
+participant ":SortCommand" as SC
+
+Main -> SC: execute()
+activate SC
+
+create participant "SortedEventList :EventList" as SEL
+SC -> SEL : new EventList(eventList)
+activate SEL
+SEL --> SC:
+deactivate SEL
+
+alt sort by name
+    SC -> SEL: sortByName()
+    activate SEL
+    SEL --> SC: return sorted list
+    deactivate SEL
+else sort by time
+    SC -> SEL: sortByTime()
+    activate SEL
+    SEL --> SC: return sorted list
+    deactivate SEL
+else sort by priority
+    SC -> SEL: sortByPriority()
+    activate SEL
+    SEL --> SC: return sorted list
+    deactivate SEL
```

Comment:
I think you don't have to state "return sorted list" for the return arrows, as it's clear that sortedEventList is already being sorted?

![image](https://github.com/user-attachments/assets/3cabdf3e-cb82-4bb5-8988-bdf2efaa2fe1)